srcpkgs/pistol/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'pistol'
+pkgname=pistol
+version=0.5.2
+revision=1
+build_style=go
+go_import_path=github.com/doronbehar/pistol
+go_package="${go_import_path}/cmd/pistol"
+hostmakedepends="ruby-asciidoctor"
+makedepends="file-devel libmagic"
```

Comment:
The `libmagic` dependency is not required, `file-devel` depends already on it (and in `makedepends` you generally only list `-devel` packages).